xc7/primitives/common_slice/Nlut/simtest/test_alut.py

```diff
@@ -1,51 +0,0 @@
-"""
```

Comment:
Did you mean to delete this file?

Reply:
Yes, cocotb is no longer a dependency we support without significant effort.  We should find another approach to testing sim.v files.

Reply:
I created #598 to come back on testing sim.v files. Is there a reason removing this should be in this PR? If we are going to delete them, I'd prefer an independent PR that removes all of them at once so we can revert to resurrect parts in the future.

Reply:
> I created #598 to come back on testing sim.v files. Is there a reason removing this should be in this PR? If we are going to delete them, I'd prefer an independent PR that removes all of them at once so we can revert to resurrect parts in the future.

In order to enable pytest on the xc7 tree, all python file imports must work.  In this case, test_alut.py fails because cocotb is no longer supported.

Reply:
ok makes sense.